task/04-arrays-tasks.js

```diff
@@ -286,16 +292,22 @@ function propagateItemsByPositionIndex(arr) {
  *   [ 10, 10, 10, 10 ] => [ 10, 10, 10 ]
  */
 function get3TopItems(arr) {
-   throw new Error('Not implemented');
+    function compare(a, b) {
+        if (a > b) return -1;
+        if (a === b) return 0;
+        if (a < b) return 1;
+    }
+
+    return arr.sort(compare).splice(0, 3);
 }
```

Comment:
there is no reason to create an extra function, you can define it right in the `sort` call
.sort((a, b) => {...})